core/templates/components/common-layout-directives/common-elements/sharing-links.component.html

```diff
@@ -18,7 +18,7 @@
   </div>
   <div *ngIf="shareType === 'exploration'" title="Embed this Exploration" class="fx-main-center fx-cross-center padding-0-2">
     <a class="fx-main-center fx-cross-center" (click)="showEmbedExplorationModal()" tabindex="0">
-      <i class="material-icons md-48 embed-link share-icons" [ngClass] = "smallFont ? 'font-small' : 'font-big'" >&#xE86F;</i>
+      <i class="material-icons embed-link share-icons emb" [ngClass] = "smallFont ? 'font-small' : 'font-big'">&#xE86F;</i>
```

Comment:
Ditto: Let's use a more descriptive name here for the class `emb`.

Reply:
okay